### PR TITLE
Disable in people picker

### PR DESCRIPTION
**Pull request by SanTran113**
made some changes to a selected person, changed submit question post call (i forgot before), hardcode disable

### CODE REVIEW

File: frontend/src/components/game/PeoplePicker.tsx [hunk 1]
```diff
@@ -47,7 +50,8 @@ const Select: React.FC<PeoplePickerProps> = ({
                     people[0],
```
Comment (StarDylan) [inline frontend/src/components/game/PeoplePicker.tsx:50]: What do you think about changing the onClick so it doesn't call onSelectPerson when disabled?

**Review by Anamooz — APPROVED** [review]
Looks good, I like the tailwind animations you added when picking a person along with the outline around the logo, makes it very easy to tell who I'm picking

**Review by StarDylan — APPROVED** [review]
Looks great!

**Review by thekevinWHYuan — APPROVED** [review]
LGTM!

**Review by nrojo1124 — APPROVED** [review]
lgtm!

**Pull request merged**